caas/kubernetes/provider/specs/admissionregistration.go

```diff
@@ -57,6 +57,71 @@ func (wh K8sMutatingWebhookSpec) MarshalJSON() ([]byte, error) {
 	}
 }
 
+// UpgradeK8sMutatingWebhookSpecV1Beta1 converts a v1beta1 MutatingWebhook to v1.
+func UpgradeK8sMutatingWebhookSpecV1Beta1(spec admissionregistrationv1beta1.MutatingWebhook) admissionregistrationv1.MutatingWebhook {
```

Comment:
I guess these upgrade funcs are implicitly tested else (eg TestEnsureValidatingWebhookConfigurationsCreateV1Beta1Upgrade) so we can punt on not doing unit tests here.

Reply:
Yeah, I'd rather not duplicate testing just because I had to move packages.